### PR TITLE
feat(util): add camelToSnakeCase() & snakeToCamelCase()

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [ ] Code changes have been tested against the Discord API, or there are no
  code changes

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed,
    parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation,
  README, etc.

### CODE REVIEW

**Comment by ayntee** [comment]
You should change the existing entities to camel case as well and not just add the functions.

File: src/util/utils.ts [hunk 1]
```diff
@@ -60,3 +60,52 @@ export const formatImageURL = (
...
+    if ($1 === "_id") return "ID";
+    return $1.toUpperCase().replace("_", "");
+  });
+}
+
+export function isObject(o: unknown) {
+  return o === Object(o) && !Array.isArray(o) && typeof o !== "function";
+}
+// deno-lint-ignore no-explicit-any
+export function camelKeysToSnakeCase(o: any) {
```
Comment (ayntee) [inline src/util/utils.ts:82]:
```suggestion
export function camelKeysToSnakeCase(o: Record<string, any>) {
```
Also, a better parameter name like `obj`.

File: src/util/utils.ts [hunk 2]
```diff
@@ -60,3 +60,52 @@ export const formatImageURL = (
...
+}
+
+export function snakeToCamelCase(s: string) {
+  return s.replace(/_id|([-_][a-z])/ig, ($1) => {
+    if ($1 === "_id") return "ID";
+    return $1.toUpperCase().replace("_", "");
+  });
+}
+
+export function isObject(o: unknown) {
```
Comment (ayntee) [inline src/util/utils.ts:78]:
```suggestion
function isObject(o: unknown) {
```
Do not export private functions

File: src/util/utils.ts [hunk 3]
```diff
@@ -60,3 +60,52 @@ export const formatImageURL = (
...
 };
+
+export function camelToSnakeCase(str: string) {
+  return str.replace(/ID|[A-Z]/g, (s) => {
+    if (s === "ID") return "_id";
+    return `_${s.toLowerCase()}`;
+  });
+}
+
+export function snakeToCamelCase(s: string) {
```
Comment (ayntee) [inline src/util/utils.ts:71]:
```suggestion
function snakeToCamelCase(s: string) {
```
Do not export private functions

File: src/util/utils.ts [hunk 4]
```diff
@@ -60,3 +60,52 @@ export const formatImageURL = (
...
+export function isObject(o: unknown) {
+  return o === Object(o) && !Array.isArray(o) && typeof o !== "function";
+}
+// deno-lint-ignore no-explicit-any
+export function camelKeysToSnakeCase(o: any) {
+  if (isObject(o)) {
+    // deno-lint-ignore no-explicit-any
+    const n: Record<string, any> = {};
+    Object.keys(o)
+      .forEach((k) => {
```
Comment (ayntee) [inline src/util/utils.ts:87]: A better name for the argument "k".

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**